src/daria/corrections/shape/drift.py

```diff
@@ -44,8 +49,47 @@ def __init__(
         else:
             raise ValueError("Data type for baseline image not supported.")
 
-        # Cache roi
-        self.roi = roi if isinstance(roi, tuple) else daria.bounding_box(roi)
+        # Cache config
+        if config is not None:
+            if isinstance(config, str):
+                with open(str(Path(config)), "r") as openfile:
+                    self.config = json.load(openfile)
+            elif isinstance(config, Path):
+                with open(str(config), "r") as openfile:
+                    self.config = json.load(openfile)
+            else:
+                self.config = copy.deepcopy(config)
+        else:
+            self.config: dict = {}
+
+        # Cache ROI
+        if isinstance(roi, np.ndarray):
+            self.roi = daria.bounding_box(roi)
+            self.config["roi_drift_correction"] = self.roi.tolist()
+        elif isinstance(roi, list):
+            self.roi = daria.bounding_box(
+                np.array(roi),
+                padding=round(0.05 * self.base.shape[0]),
+                max_size=[self.base.shape[0], self.base.shape[1]],
+            )
+            self.config["roi_drift_correction"] = daria.bounding_box_inverse(
+                self.roi
+            ).tolist()
+        elif isinstance(roi, tuple):
+            self.roi = roi
+            self.config["roi_drift_correction"] = daria.bounding_box_inverse(
+                roi
+            ).tolist()
+        elif "roi_drift_correction" in self.config:
```

Comment:
We should consider having some txt file which gives an overview over all keywords in case on build such a config file from scratch or modifies an existing one. Some explanations at some place would be great.

Reply:
Thinking of my comment on nested config files, I would have not organized it like this, but rather check for a more general and simple keyword "roi" which is contained in a dictionary with keyword "drift" e.g.